docs/pages/User/GettingStarted/further-steps.md

```diff
@@ -0,0 +1,77 @@
+checkoutservice-779cb9bfdf-h48tg         1/1     Running   0          13m   172.16.97.201    liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+currencyservice-5db6c7d559-gb7ln         1/1     Running   0          12m   172.16.226.110   liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+emailservice-5c47dc87bf-zzz4z            1/1     Running   0          13m   10.244.2.235     worker-node-2
+frontend-5fcb8cdcdc-vvq4m                1/1     Running   0          13m   172.16.97.207    liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+loadgenerator-79bff5bd57-t7976           1/1     Running   0          12m   172.16.97.208    liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+paymentservice-6564cb7fb9-vn7pn          1/1     Running   0          13m   172.16.97.197    liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+productcatalogservice-5db9444549-cxjlb   1/1     Running   0          13m   172.16.226.87    liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+recommendationservice-78dd87ff95-2s8ks   1/1     Running   0          13m   10.244.2.241     worker-node-2
+redis-cart-57bd646894-9x4cd              1/1     Running   0          12m   172.16.226.120   liqo-9a596a4b-591c-4ac6-8fd6-80258b4b3bf9
+shippingservice-f47755f97-5jcpm          1/1     Running   0          12m   10.244.4.169     worker-node-1
+```
+
+## Access the demo application
+
+Once the deployment is completed, you can start using the demo application and verify that everything works correctly even if its components are distributed across multiple Kubernetes clusters.
+By default, the frontend web-page is exposed through a `LoadBalancer` service, which can be inspected using:
+```bash
+kubectl get service -n liqo-demo frontend-external
+```
+
+You should see an output similar to the following, indicating that you can access the demo application at `http://xxx.yyy.zzz.www`:
+```
+NAME                TYPE           CLUSTER-IP    EXTERNAL-IP       PORT(S)        AGE
+frontend-external   LoadBalancer   10.96.8.220   xxx.yyy.zzz.www   80:32233/TCP   15m
+```
+
+In case the `EXTERNAL-IP` field is characterized by a `<pending>` value, on the other hand, you are probably using a bare metal cluster (i.e. not managed by an IaaS platform such as GCP, AWS, Azure, ...) and lacking a Load Balancer implementation.
+
+Still, you have two alternatives to access the demo application:
+1. Leverage the `NodePort` (e.g. 32233 in the previous example), which can be accessed using the IP address of any of the physical servers of your home cluster;
+2. Leverage `kubectl port-forward` to forward the requests from your local machine (i.e. `http://localhost:8080`) to the frontend service:
+   ```bash
+   kubectl port-forward -n liqo-demo service/frontend-external 8080:80
+   ```
+
+> **Clean-up**: If you want to delete the deployed example, just issue:
+> ```bash
+> kubectl delete -f https://github.com/liqotech/microservices-demo/blob/master/release/kubernetes-manifests.yaml -n liqo-demo
+> ```
```

Comment:
Wouldn't be possible to kill some pods, see if it is scheduled on the other cluster, and see that everything works smoothly? This would be an impressive demo for me.

Reply:
We can't force rescheduling on a specific node, but we can definitely show that upon deletion the pod is rescheduled.